legal-api/report-templates/amalgamationApplication.html

```diff
@@ -1,13 +1,48 @@
 [[macros.html]]
-
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-      <title>Amalgamation Application</title>
-      <meta charset="UTF-8">
-      <meta name="author" content="BC Registries and Online Services">
-      [[common/style.html]]
-    </head>
-    <body>
-    </body>
+  <head>
+    <title>Amalgamation Application</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="BC Registries and Online Services">
+    [[common/style.html]]
+  </head>
+  <body>
+    <div class="header">
+      <table class="header-table" role="presentation">
+        <tr class="no-page-break">
+          <td>
+            [[logo.html]]
+          </td>
+          <td>
+            <div class="report-type">AMALGAMATION APPLICATION</div>
+            <div class="report-type-desc">{{ entityDescription }} - {{ entityAct }}</div>
+          </td>
+        </tr>
+      </table>
+      <div class="business-name-header">
+        <label class="lbl-business-name">{{nameRequest.legalName}}</label>
+      </div>
+    </div>
+    [[common/businessDetails.html]]
+    <div class="no-page-break">
+      <div class="section-header mt-5 pt-4">AMALGAMATION APPLICATION</div>
+    </div>
+    [[amalgamation/nameRequest.html]]
+    [[amalgamation/effectiveDate.html]]
+    [[amalgamation/approvalType.html]]
+    [[amalgamation/amalgamatingCorp.html]]
+    [[amalgamation/amalgamationStmt.html]]
+    [[amalgamation/completingParty.html]]
+    [[amalgamation/completingPartyStmt.html]]
```

Comment:
this can be replaced with [[incorporation-application/completingParty.html]] (Completing Party Statement is handled inside this file)

Reply:
The incorporation-application does not have delivery address, I can either add it there or keep the amalgamation.

Reply:
You can keep this one. Although ideally it should be same as incorporation-application. Can you check this with Yui?

Reply:
Checked with Yui. It was an error in the design. Will be going back to the incorporation-application version.